src/ProfiledAot/src/recordTrace.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
```

Comment:
I normally run this on Windows, is there a way this could be all in MSBuild (using `<Exec/>`, etc.) so it could work on both Mac & Windows?

Reply:
I haven't managed to run two `<Exec/>` in parallel, but if there is a way to start one (dotnet-dsrouter) in the background, and have msbuild continue to the next exec (dotnet-trace) while the first one is still running, that should work as long as the users don't already have other dotnet-dsrouter sessions ongoing

Reply:
It sounds like we need a custom MSBuild task that launches these, maybe we need to implement a target in the Android workload?